index.html

```diff
@@ -54,16 +54,225 @@ <h1 class="subSentenceText">I'm a Software Developer</h1>
           </a>
         </li>
       </ul>
+      <a href="#" class="background">
+        <img src="background.png" alt="Background Image" />
+      </a>
     </section>
+
+    <!-- lower section of my portfolio template -->
+
+    <section>
+      <div class="SecondPage">
+        <div>
+          <h1 class="MyRecentWorksTop">My Recent Works</h1>
+          <hr class="AboutLine1">
+        </div>
+        <div>
+          <a href="#" class="mainImage">
+          <img src="mainImage.png" alt="Image" />
+          </a>
+          <h2 class="multiPostStories">Multi-Post Stories</h2>
+          <p class="pageInformation">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="techStacks">
+            <li>
+              <button class="stackButton">CSS</button>
+            </li>
+            <li>
+              <button class="stackButton">HTML</button>
+            </li>
+            <li>
+              <button class="stackButton">Bootstreap</button>
+            </li>
+            <li>
+              <button class="stackButton">Ruby</button>
+            </li>
+          </ul>
+          <button class="firstProjectButton">See Project</button>
+        </div>
+        <!-- these are my pages container -->
+        <div class="ThirdPage">
+          <div class="Screen1">
+            <h3 class="PrintingData">Professional Art Printing Data</h3>
+            <p class="textSegment">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="builtWith">
+              <li>
+                <button class="buttonOfTechStack">HTML</button>
+              </li>
+              <li>
+                <button class="buttonOfTechStack">Bootstrap</button>
+              </li>
+              <li>
+                <button class="buttonOfTechStack">Ruby</button>
+              </li>
+            </ul>
+            <button class="secondProjectButton">See Project</button>
+          </div>
+
+          <div class="Screen2">
+            <h3 class="PrintingData">Professional Art Printing Data</h3>
+            <p class="textSegment">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="builtWith">
+              <li>
+                <button class="buttonOfTechStack">HTML</button>
+              </li>
+              <li>
+                <button class="buttonOfTechStack">Bootstrap</button>
+              </li>
+              <li>
+                <button class="buttonOfTechStack">Ruby</button>
+              </li>
+            </ul>
```

Comment:
- [ ] Nicw work so far 👍🏿 implement your items but I kindly suggest you remove the `button` elements and use only the `ul`` li `tags to lay out your items 😉

Reply:
- [ ]  Kindly let's implement the suggestion from the previous reviewer. 🙏🏽 It would enhance the quality of your code 😁 😁  You can achieve that by following the instruction I mentioned above ☝🏽 ☝🏽